### PR TITLE
Stricter ESlint with Prettier

### PR DESCRIPTION
**Pull request by yjkimjunior**
Mainly:
* ~spaces => tabs~
* no unused vars

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
   "endOfLine": "lf",
-  "semi": false,
-  "singleQuote": false,
+  "jsxSingleQuote": true,
+  "semi": true,
+  "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "useTabs": true
```
Comment (amaurymartiny) [inline .prettierrc]: bold move!

File: .eslintrc.js [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
-  extends: `react-app`,
-}
\ No newline at end of file
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'semistandard',
+    'prettier',
+    'plugin:prettier/recommended',
+    'plugin:import/typescript',
```
Comment (amaurymartiny) [inline .eslintrc.js]: I was actually talking about https://github.com/lydell/eslint-plugin-simple-import-sort#usage, not eslint-plugin-import (which afaict does not sort imports).

File: .prettierrc (hunk 1 above)
Comment (yjkimjunior) [reply .prettierrc]: it had to be done

File: package.json [hunk 3]
```diff
@@ -18,8 +18,13 @@
     ]
   },
   "devDependencies": {
+    "@polkadot/dev": "^0.31.1",
+    "eslint-config-prettier": "^6.6.0",
     "eslint-config-react-app": "^5.0.2",
```
Comment (amaurymartiny) [inline package.json]: We can remove this one then

**Pull request merged**